mesh2hrtf/Mesh2Input/mesh2input.py

```diff
@@ -318,6 +329,10 @@ def save(operator,
         if os.path.exists(os.path.join(filepath1, "NumCalc")):
             raise ValueError((f"Project folder {filepath1} already exists. "
                               "Choose another folder or delete files."))
+        if checkall:
+            _write_checkall_txt(filepath1) #Added by Sanket
+
```

Comment:
I would suggest to move this up, where we test for a triangular mesh. It might even make sense to include rthe triangular check only if the user wants to do the checks. Can you benchmark how long those checks take? This would help to decide if it would make sense to include separate parameters for different checks.